src/pages/ALLO_ManageAllocations.page

```diff
@@ -197,7 +197,7 @@
                                     </div>
                                 </td>
                                 <td>
-                                    <apex:commandButton styleClass="slds-button slds-button_destructive" value="{!$Label.alloDeleteRow}" id="delRowBTN" title="{!$Label.alloDeleteRow}" action="{!delRow}" reRender="theTable" immediate="true">
+                                    <apex:commandButton styleClass="slds-button slds-button_destructive" value="{!$Label.alloDeleteRow}" id="delRowBTN" title="{!$Label.alloDeleteRow}" action="{!delRow}" reRender="theForm" immediate="true">
```

Comment:
@jjbennett why this change? The pagemessage is outside of the table but in the form?

Reply:
@njjc yes, that is correct.

Reply:
@njjc I don't think we need your fix for the % bug, I think reloading the form vs the table might have fixed it already?

Reply:
Interesting, let's meet later today and discuss!